README.md

```diff
@@ -5,8 +5,7 @@
 [![License](http://img.shields.io/:license-Apache%202-blue.svg)](LICENSE)
 [![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://bit.ly/2tFsN1X)
 
-Amundsen is a data discovery portal to make data more discoverable. The project named after Norwegian explorer [Roald Amundsen](https://en.wikipedia.org/wiki/Roald_Amundsen) collects and displays
-metadata about various data resources such as Hive and Redshift tables.
+Amundsen is a metadata driven application for improving the productivity of data analysts, data scientists and engineers when interacting with data. It does that today by indexing data resources (tables, dashboards, streams, etc.) and powering a page-rank style search based on usage patterns (e.g. highly queried tables show up earlier than less queried tables). Think of it as Google search for data. The project is named after Norwegian explorer [Roald Amundsen](https://en.wikipedia.org/wiki/Roald_Amundsen), the first person to discover South Pole.
```

Comment:
I added something similar previously. I think @ttannis 's concern is that we have dashboards on our roadmap, but not yet supported so far(same as streams).

Reply:
But I am ok with that, and we could update the roadmap doc(or Mark if you have time) to include the upcoming features.